docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -0,0 +1,64 @@
+[role="xpack"]
+[[put-search-application]]
+=== Put Search Application
+
+++++
+<titleabbrev>Put Search Application</titleabbrev>
+++++
+
+Creates or updates a Search Application.
+
+[[put-search-application-request]]
+==== {api-request-title}
+
+`PUT _application/search_application/<name>`
+
+[[put-search-application-prereqs]]
+==== {api-prereq-title}
+
+Requires the `manage_search_application` cluster privilege.
+Also requires <<privileges-list-indices,manage privileges>> on all indices that are added to the Search Application.
+
+[[put-search-application-path-params]]
+==== {api-path-parms-title}
+
+`create`::
+(Optional, Boolean) If `true`, this request cannot replace or update existing Search Applications.
+Defaults to `false`.
```

Comment:
We decided to default to `false` because this call will also be used regularly to update existing search applications. Furthermore the behavior was consistent with `create` flags in many other calls such as the [Indices Put Template](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-template.html) call.